mautic/mautic
# File import enhancement for contact import

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [N]
| New feature/enhancement? (use the a.x branch)      | [Y]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [N]
| Automated tests included?              | [Y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Create a separate import directory for lead import and add header 
`header("Connection: close");` to support large imports
https://www.darrenhorrocks.co.uk/739/php-problem-with-upload_err_partial-file-upload-error-code-3/
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Import contact csv and verify it is uploaded in correct directory.
3. Also try with large file so that we can see no error for import.
